zigpy/device.py

```diff
@@ -79,7 +79,11 @@ def name(self) -> str:
 
     @property
     def non_zdo_endpoints(self) -> list[zigpy.endpoint.Endpoint]:
-        return [ep for epid, ep in self.endpoints.items() if epid != 0]
+        return [
+            ep
+            for epid, ep in self.endpoints.items()
+            if epid != 0 and not (isinstance(ep, zdo.ZDO))
```

Comment:
isn't this redundant? endpoint id is reserved for ZDO endpoints

Reply:
I think so, but I do not know all "secrets" ... so I left the original condition in place.
'mypy' does not understand that 'epid!=0' means its an Endpoint instance.

Ok if I remove the !=0 test?

Reply:
I removed the epid!=0, tests pass.